### PR TITLE
Manage jstoolbar translations with I18n

### PR DESCRIPTION
**Pull request by tessi**
We have vendored jstoolbar translation as separate javascript files.

This PR aims to manage the jstoolbar translations with regular I18n to make them equal with all our other translations. This lets us add jstoolbar translations with crowdin \o/

### CODE REVIEW

File: lib/redmine/wiki_formatting/textile/helper.rb [hunk 1]
```diff
@@ -45,10 +45,32 @@ def initial_page_content(page)
...
+            'Image'             => ::I18n.t('wiki_formatting.image'),
+          }
+        end
+
         def heads_for_wiki_formatter
           unless @heads_for_wiki_formatter_included
             content_for :header_tags do
-              javascript_include_tag("jstoolbar/lang/jstoolbar-#{current_language.to_s.downcase}")
+              javascript_tag do
+                "jsToolBar.strings = #{jstoolbar_translations.to_json};".html_safe
```
Comment (mfrister) [inline lib/redmine/wiki_formatting/textile/helper.rb]: I'm pretty much opposed to adding any inline JavaScript as I hope we'll one day be able to disable inline scripts via content security policy.

Couldn't we use i18n-js for this? (Assuming that's still our up-to-date method of giving translations to JavaScript. If not, we're probably already using another method for Angular.)

Comment (myabc) [inline lib/redmine/wiki_formatting/textile/helper.rb]: @meeee we're still using i18n-js (which I'm not exactly thrilled about)

Comment (tessi) [inline lib/redmine/wiki_formatting/textile/helper.rb]: I see. Let me see if I can use i18n-js for that

**Comment by tessi** [comment]
Now, I18n-js is used to get the translations.

File: lib/redmine/wiki_formatting/textile/helper.rb [hunk 2]
```diff
@@ -46,12 +46,6 @@ def initial_page_content(page)
         end
 
         def heads_for_wiki_formatter
```
Comment (mfrister) [inline lib/redmine/wiki_formatting/textile/helper.rb:54]: Is there a reason to keep this empty method around?

Comment (tessi) [inline lib/redmine/wiki_formatting/textile/helper.rb:54]: That's the API of those formatting helpers. See: https://github.com/opf/openproject/blob/dev/lib/redmine/wiki_formatting/null_formatter/helper.rb#L37

Comment (tessi) [inline lib/redmine/wiki_formatting/textile/helper.rb:54]: I do not intend to change the formatting helper API in this PR

**Comment by mfrister** [comment]
:+1:

**Pull request merged**